crates/router/src/connector/utils.rs

```diff
@@ -78,6 +78,8 @@ pub fn get_unimplemented_payment_method_error_message(connector: &str) -> String
     format!("Selected payment method through {}", connector)
 }
 
+pub const UNSUPPORTED_PAYMENT_METHOD_ERROR_MESSAGE: &str = "Selected payment method";
```

Comment:
rename this to SELECTED_PAYMENT_METHOD and use it in `get_unimplemented_payment_method_error_message` method